alf/utils/common.py

```diff
@@ -999,14 +999,14 @@ def exe_mode_name():
 
 def is_replay():
     """Return a bool value indicating whether the current code belongs to
-    unrolling or training.
+    replaying. Replaying implies off-policy training.
```

Comment:
I think we need clearer definition of which part of code belongs to replaying. Similar for is_rollout and is_training.

Reply:
> I think we need clearer definition of which part of code belongs to replaying. Similar for is_rollout and is_training.

Do you mean clearer definition in terms of code or docstring comment?

Reply:
Describe more clearly what part of code belongs to replay in the doc-string

Reply:
Although that the purpose of each function is clear, the definition of ``replay/unroll/rollout(unroll)`` here is a bit different from other places in ALF, with similar notions, e.g. when summarizing time cost.
This could introduce some confusions. It might be best to keep these terms to have consistent meanings at different places.

Reply:
> Although that the purpose of each function is clear, the definition of `replay/train/rollout(unroll)` here is a bit different from other places in ALF, with similar notions, e.g. when summarizing time cost. This could introduce some confusions. It might be best to keep these terms to have consistent meanings at different places.

The purpose of these functions is quite different from other places. They are mainly used as flags for controlling different logics in the same code block to generate different behaviors. If aligning with the terminology of time cost summary, then they can no longer serve this purpose. For example, "time/train" only summarizes ``train_from_unroll()`` for on-policy algorithms. Generally the the labels for time cost are more fine-grained.